### ISSUE TITLE
Check if page numbers are consistent between PDF and EPUB outputs

### ISSUE THREAD

**Issue by chrisdaaz** [nulib-oer/lantern#36]
Generate EPUB and PDF document examples and see if the page numbers are same between both formats. Try a few examples, with many and few numbers of chapters, to note any differences.

**Comment by AngelinaJaglinski** [comment]
@chrisdaaz I've generated some examples, but I'm having trouble finding the page numbers for the epub format.  I tried using iBooks and Calibre as epub readers.  In both cases, page numbers don't appear on the "page" itself.  Out of the two, only iBooks has page numbers, but these change depending on the size of your window.

Should I be able to see the page numbers written on the pages of the epub file?  Or is there a different epub reader that has consistent page numbers regardless of window size?  Thanks!
